backend/yugioh/models.py

```diff
@@ -0,0 +1,80 @@
+from django.db import models
+
+from yugiohcardinset.models import YugiohCardInSet
+
+
+class YugiohCard(models.Model):
+
+    type = models.CharField(
+        max_length=100,
+    )
+
+    frame_type = models.CharField(
+        max_length=100,
+    )
+
+    description = models.CharField(
+        max_length=1000,
+        blank=True,
+        null=True,
+    )
+
+    attack = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    defense = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    level = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    race = models.CharField(
+        max_length=100,
+    )
+
+    attribute = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    archetype = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    image = models.URLField(
+        blank=True,
+        null=True,
+    )
+
+    card = models.ForeignKey(
+        'card.Card',
+        on_delete=models.CASCADE,
+    )
+
+    def __str__(self):
+        return self.card
+
+
+class YugiohCardYugiohCardInSet(models.Model):
```

Comment:
- [ ] Rename to `YugiohCardInSet`

Reply:
We already have YugiohCardInSet model

Reply:
YugiohCard and YugiohCardInSet have been added to admin panel.